public/data/CBID2.json

```diff
@@ -24,5 +24,13 @@
       "description": "My first award on GitHub",
       "url": "https://github.com/users/CBID2/achievements/pull-shark"
     }
+    {
+      "title": "SheSharp New in Tech Scholarship Recepient",
+      "date":"October 2022",
+      "icon": "github",
+      "color": "grey",
+      "description":"Shesharp is a non-profit foundation that is dedicated to helping women and non-binary people start their careers in tech. I received this award as a result of expressing my interest in bridging the gaps between instructional design and coding in addtion to being an active member in SheSharp's Slack community.",
+      "url": "https://www.linkedin.com/posts/shesharp_announcing-our-new-in-tech-scholarship-winner-activity-6993866371780476928-q5-3?utm_source=share&utm_medium=member_desktop",
+    }
```

Comment:
Please see the GitHub Action inline comments about the invalid json

Also json file change should go to `main` branch (but the README changes should be to the `nextjs` branch)

Reply:
Ok. How do I make sure the README file changes go into the mainJS?

Reply:
create a new branch from the default branch for each separate change

Reply:
Still lost